ONSdigital/dp-frontend-renderer
# Feature/cookies preferences success banner

### What

- Updated cookies preferences page markup to include success message to be shown on successful edit action
- Updated localisation files to include success text
- Updated cookies handler to accept new cookiesPreferences page model

### How to review

- Check that code makes sense
- Go to `/cookies` and test that the success message is displayed when updating cookies preferences

### Who can review

Anyone but me
